specs/Economics/README.md

```diff
@@ -16,20 +16,21 @@
 | Name | Value |
 | - | - |
 | `INITIAL_SUPPLY` | `10**33` yoctoNEAR |
-| `NEAR` | `10**24` yoctoNEAR |
 | `MIN_GAS_PRICE` | `10**5` yoctoNEAR |
 | `REWARD_PCT_PER_YEAR` | `0.05` |
-| `BLOCK_TIME` | `1` second |
 | `EPOCH_LENGTH` | `43,200` blocks |
 | `EPOCHS_A_YEAR` | `730` epochs |
-| `POKE_THRESHOLD` | `500` blocks |
 | `INITIAL_MAX_STORAGE` | `10 * 2**40` bytes == `10` TB |
 | `TREASURY_PCT` | `0.1` |
 | `TREASURY_ACCOUNT_ID` | `treasury` |
 | `CONTRACT_PCT` | `0.3` |
 | `INVALID_STATE_SLASH_PCT` | `0.05` |
 | `ADJ_FEE` | `0.001` |
 | `TOTAL_SEATS` | `100` |
+| `ONLINE_THRESHOLD_MIN` | `0.9` |
+| `ONLINE_THRESHOLD_MAX` | `0.99` |
+| `BLOCK_PRODUCER_KICKOUT_THRESHOLD` | `0.9` |
+| `CHUNK_PRODUCER_KICKOUT_THRESHOLD` | `0.6` |
```

Comment:
@bowenwang1996 @SkidanovAlex feel free to mention here what are these numbers should be at this point.